schema/codelists/statementType.csv

```diff
@@ -1,4 +1,4 @@
-code,title,description,technical note
-personStatement,Person Statement,,The parent object should be validated using the person-statement schema.
-entityStatement,Entity Statement,,The parent object should be validated using the entity-statement schema
-ownershipOrControlStatement,Ownership or Control Statement,,The parent object should be validated using the ownership-or-control-statement schema. 
+code,title,description,technical note,
+personStatement,Person Statement,"The statement contains information known about a natural person at a particular point in time, or from a given submission of information.",The parent object should be validated using the Person Statement schema.,
+entityStatement,Entity Statement,"The statement contains information that identifies and describes an entity at a particular point in time, or from a given submission of information.",The parent object should be validated using the Entity Statement schema.,
+ownershipOrControlStatement,Ownership or Control Statement,"The statement contains details of the interests held by an interested party in an entity at a particular point in time.",The parent object should be validated using the Ownership-or-control Statement schema.,
```

Comment:
Can we change the title of this code to 'Ownership-or-control Statement' please.